docs/development/services.md

```diff
@@ -0,0 +1,76 @@
+# Services
+## Network Diagram
+![Network Diagram](network_diagram.png)
+
+
+## Grapl Services - main pipeline
+Unless otherwise specified, the input to a service is the output of the one described above it.
+
+
+### Graph Generator
+**Input:** Event logs (i.g. sysmon logs, osquery logs, Cloudtrail logs) from the customer's S3 bucket.
+
+**Work:** Turns these events into a standalone subgraph, independent of existing Dgraph state.
+
+**Output:** Stores the subgraph to S3 for Node Identifier.
+
+
+### Node Identifier
+**Work:** Identifies nodes in the incoming subgraph against DynamoDB. 
```

Comment:
I'm still coming to grips w/ the arch so pardon me if I'm off base -- DynamoDB holds the _canonical identities_ of the nodes?  is that the term we use to distinguish IDs in the incoming subgraph vs. the IDs in the master graph?

Reply:
I'm not 100% sure myself, but that's my understanding - the dynamodb tables help deal with deduplication and correlation. 

(Ideally, these will be fleshed out or rewritten by experts.)